### ISSUE TITLE
Persoons Beheer Huiswerkopdracht

### ISSUE THREAD

**Issue by gglerum** [DesoP7/Thunder#2]
Shamir, de huiswerk opdracht voor het beheren van personen zie ik niet staan, zie: [PersoonsBeheer](https://github.com/MKB-Cyber-Campus/java-lessen/blob/master/02%20-%20Basics/students.md). Mocht je die vergeten zijn te pushen dan hoor ik dat graag.

Mocht je deze nog niet hebben gemaakt, kijk dan naar het Huiswerk van deze week, dat zijn namelijk aanpassingen voor die PersoonsBeheer opdracht. Dan kun je de wijzigingen die je anders zou moeten maken meteen doorvoeren.

**Comment by DesoP7** [comment]
Ja, je zei als het lukt hoef ik maar 1 te sturen, maar ik ging net door de opdracht van dit week en zag dat ik die van vorige week nodig heb. dus zal ik het met de wijzigingen maken voordat ik hem gaat pushen
